webapp/src/ts/services/form.service.ts

```diff
@@ -351,3 +352,40 @@ export class FormService {
   }
 }
 
+export class WebappEnketoFormContext implements EnketoFormContext {
```

Comment:
Nice moving this to forms service instead of enketo service! I was very tempted to do the same thing in a PR of mine, but it was too much an unrelated change. Thanks for making this switch!